Doc/library/sqlite3.rst

```diff
@@ -309,21 +318,24 @@ Module functions and constants
       Added the ``sqlite3.connect/handle`` auditing event.
 
 
-.. function:: register_converter(typename, callable)
+.. function:: register_converter(type, callable)
 
-   Registers a callable to convert a bytestring from the database into a custom
-   Python type. The callable will be invoked for all database values that are of
-   the type *typename*. Confer the parameter *detect_types* of the :func:`connect`
-   function for how the type detection works. Note that *typename* and the name of
-   the type in your query are matched in case-insensitive manner.
+   Register *callable* to convert SQLite *type* into a Python type.
+   *callable* is invoked for all values of type *typename*.
+   Confer the parameter *detect_types* of :meth:`Connection.connect`
+   regarding how type detection works.
+
+   .. note::
+      *type* and the name of the type in your query are matched in a
+      case-insensitive manner.
 
 
 .. function:: register_adapter(type, callable)
 
-   Registers a callable to convert the custom Python type *type* into one of
-   SQLite's supported types. The callable *callable* accepts as single parameter
-   the Python value, and must return a value of the following types: int,
-   float, str or bytes.
+   Register *callable* to adapt Python *type* into an SQLite type.
```

Comment:
My English is poor, but the former wording looks more grammatically correct to me. Emphasized `*callable*` and `*type*` are names of parameters. Either `callable *callable*` or `a callable`.

Reply:
I'll come up with something. Thanks!

Reply:
Reworded in 82cf3e2ff5